app/assets/javascripts/discourse/app/mixins/uppy-s3-multipart.js

```diff
@@ -1,13 +1,13 @@
 import Mixin from "@ember/object/mixin";
+import { bind } from "discourse-common/utils/decorators";
 import { Promise } from "rsvp";
 import { ajax } from "discourse/lib/ajax";
 import AwsS3Multipart from "@uppy/aws-s3-multipart";
 
 export default Mixin.create({
   _useS3MultipartUploads() {
     this.set("usingS3MultipartUploads", true);
-    const self = this;
-    const retryDelays = [0, 1000, 3000, 5000];
+    this.set("retryDelays", [0, 1000, 3000, 5000]);
```

Comment:
is this initialised to null somewhere ?

Reply:
btw. I think that could be a constant

Reply:
yes I don't see this being altered later on ?